Sources/Send/StateMachine.swift

```diff
@@ -4,13 +4,12 @@
 
 import Foundation
 
-public protocol StateMachine<State, Action, Services> {
-    associatedtype State
-    associatedtype Action
-    associatedtype Services
-    
-    var services: Services { get }
```

Comment:
I think `services` is optional, StateMachine at the moment only cares about `State` and `Action`.